### PR TITLE
Addition of ep detection mode for mso_schema_template_bd (DCNE-64)

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

**Review by gmicol — DISMISSED** [review]
LGTM

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@
       validate_certs: '{{ mso_validate_certs | default(false) }}'
       use_ssl: '{{ mso_use_ssl | default(true) }}'
       use_proxy: '{{ mso_use_proxy | default(true) }}'
-      output_level: '{{ mso_output_level | default("info") }}'
+      output_level: '{{ mso_output_level | default("debug") }}'
```
Comment (sajagana) [inline tests/integration/targets/mso_schema_template_bd/tasks/main.yml:24]: Is the output_level set to `debug` for a reason?

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml [hunk 2]
```diff
@@ -405,13 +405,31 @@
       name: VRF
       schema: '{{ mso_schema | default("ansible_test") }}'
       template: Template1
+    intersite_bum_traffic: true
+    arp_flooding: true
+    ep_move_detection_mode: true
   register: nm_add_bd_4
 
+- name: Add bd 4 (normal mode) with no ep_move_detection_mode
```
Comment (akinross) [inline tests/integration/targets/mso_schema_template_bd/tasks/main.yml:413]: is this not an update operation?
```suggestion
- name: Update bd 4 with ep_move_detection_mode to False
```

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml [hunk 3]
```diff
@@ -448,6 +466,8 @@
     - nm_add_bd_2.current.subnets[3].scope == "private"
     - nm_add_bd_2.current.subnets[3].shared == false
     - nm_add_bd_2.current.subnets[3].querier == false
+    - nm_add_bd_4.current.epMoveDetectMode == "garp"
+    - nm_add_bd_4_no_ep.current.epMoveDetectMode == "none"
```
Comment (akinross) [inline tests/integration/targets/mso_schema_template_bd/tasks/main.yml:470]: should we add previous for this update case?

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml [hunk 4]
```diff
@@ -405,13 +405,31 @@
...
+      template: Template1
+    ep_move_detection_mode: false
+  register: nm_add_bd_4_no_ep
+
 - name: Verify nm_add_bd_2 and nm_add_bd_3
   ansible.builtin.assert:
     that:
     - nm_add_bd_2 is changed
     - nm_add_bd_3 is changed
+    - nm_add_bd_4 is changed
```
Comment (akinross) [inline tests/integration/targets/mso_schema_template_bd/tasks/main.yml:431]: for nm_add_bd_2 and nm_add_bd_3, should we assert the default?

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml (hunk 1 above)
Comment (shrsr) [reply tests/integration/targets/mso_schema_template_bd/tasks/main.yml:24]: Yeah, I was printing debug statements in the code.

File: tests/integration/targets/mso_schema_template_bd/tasks/main.yml (hunk 4 above)
Comment (shrsr) [reply tests/integration/targets/mso_schema_template_bd/tasks/main.yml:431]: As discussed, will leave this as is

**Review by akinross — APPROVED** [review]
LGTM

**Review by sajagana — APPROVED** [review]
LGTM!

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**